express/package.json

```diff
@@ -18,36 +18,37 @@
   },
   "dependencies": {
     "@aws-sdk/client-cognito-identity-provider": "^3.80.0",
+    "@aws-sdk/client-sns": "^3.162.0",
```

Comment:
Can you please undo the changes to rearrange this file. The change should just be one line to add `"@aws-sdk/client-sns"`

Reply:
Sure.

Reply:
Done.